rails-time-track/app/controllers/api/users_controller.rb

```diff
@@ -1,13 +1,50 @@
 module Api
   class Api::UsersController < ApplicationController
+    before_action :set_user, only: [:show, :update, :destroy]
 
     def index
       render json: User.all
     end
 
+    def create
+      user = User.new(user_params)
+      if user.save
+        render json: user, status: :created
+      else
+        render json: { errors: user.errors}, status: :unprocessable_entity
+      end
+    end
+
+    def show 
+      render json: @user
+    end
+
+    def update
+      if @user.update(user_params)
+        render json: @user, status: :ok
+      else
+        render json: {errors: user.errors}
```

Comment:
The same comment as in this : https://github.com/codeableorg/TimeTrack/pull/36/files#r307050324